recipes/femto/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "femto" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Psivant/femto/archive/{{ version }}.tar.gz
+  sha256: fb367529ceb9396ffb6c5fecc8e46d8cd8e37cfd872cc34e3b6e43d986f88897
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+    - versioneer
+  run:
+    - python >=3.10
+    - numpy
+    - scipy
+    - pydantic >=2
+    - click
+    - cloup
+    - pyarrow
+    - tqdm
+    - pyyaml
+    - pandas
+    - omegaconf
+    - gputil
+
+    - sympy  # evaluating ctx param expressions
+
+    - pymbar >=4
+
+    - mpi4py
+
+    - networkx
+    - mdtraj
+
+    - ambertools
+    - openmm >=8.1.0
+
+    - seaborn-base
+    - tensorboardx
```

Comment:
Why the empty lines? It is not a problem but it makes the recipe a bit hard to track when diffs start to show up later on.

Reply:
ah that definitely makes sense! there's no real reason other than i grouped them like this in the main repo and forgot to remove them when copying over here!

should be removed in aa8c873